main/create_simulated_body_model.py

```
@@ -0,0 +1,1085 @@
+                self._offset_v_spin.setValue(self.sim_params['offset_v'])
+                self._offset_u_spin.setValue(self.sim_params['offset_u'])
+                # Rebuild tabs
+                self._rebuild_dynamic_tabs()
+                self._update_tab_titles()
+                self._validate_ranges()
+                self._updater.immediate_update()
+            except Exception as e:
+                QMessageBox.critical(self, 'Error', f'Failed to load parameters:\n{str(e)}')
+
+    # ---- Visual toggles ----
+    def _toggle_visual_aids(self, state: Any) -> None:
```

Ruff (0.14.5)

Dynamically typed expressions (typing.Any) are disallowed in `state`

(ANN401)